styles/components/HackathonModal.module.scss

```diff
@@ -0,0 +1,95 @@
+// Import fonts
+@import url('https://fonts.cdnfonts.com/css/motorblock');
+
+.Modal {
+  @apply w-11/12 max-w-2xl outline-none overflow-hidden;
+  @apply bg-opacity-0;
+}
+
+.modalImageContainer {
+  @apply absolute w-52 h-72 z-40;
+  @apply top-8 left-1/2;
+  @apply animate-hackathonModalFloat;
+
+  @media (max-width: 768px) {
+    @apply left-1/3 -top-[12.5%];
+  }
+}
+
+.modalContent {
+  @apply fixed z-50;
+  @apply top-1/2 left-1/2;
+  @apply transform -translate-x-1/2 -translate-y-[40%];
+  @apply rounded-2xl shadow-md;
+  @apply p-8;
+  @apply w-11/12 max-w-2xl;
+  @apply aspect-[1.5];
+  @apply outline-none;
+  background: linear-gradient(
+    135deg,
+    #210535 30%,
+    #102542 50%,
+    #142043 60%,
+    #1c1645 70%
+  );
+  @apply flex flex-col items-center justify-center;
+  @apply text-center;
+
+  h3 {
+    @apply text-3xl font-bold mb-2 text-color-2;
+    @apply font-semibold;
+  }
+
+  h2 {
+    @apply mb-0 text-color-2 flex flex-col items-center;
+    font-family: 'motorblock';
+    span {
+      @apply text-6xl font-bold;
+      @apply tracking-[0.2em];
+      @apply leading-none;
+      @apply p-0 m-0;
+    }
+  }
+
+  span {
+    @apply font-bold mb-6 text-color-2;
+  }
+
+  p {
+    @apply mx-auto;
+    @apply max-w-md;
+    @apply text-center;
+    @apply text-lg mb-4 text-color-2 w-1/2;
+    @apply mt-4;
+  }
+
+  .imageContainer {
+    @apply absolute inset-0;
+    @apply grid grid-cols-3 grid-rows-3;
```

Comment:
what if instead of making this a 3 cell grid you made it a 2 cell grid and used gap to space them out? I think you could also achieve a similar effect with `justify-content`. The reason I dont like this approach is it is very verbose. (If the gap approach is giving you trouble though or you think this is a better idea np).